service/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN npm install && npm run build
 # SERVICE
 #################################################
 
-FROM python:3.8-buster
+FROM python:3.9.10
```

Comment:
We zouden hier de .10 kunnen weghalen, zodat we updates meekrijgen. Volgens mij geeft Github dat niet automatisch aan en zouden we dus zelf moeten checken als we de minor version pinnen.